database-enclave/README.md

```diff
@@ -0,0 +1,58 @@
+## Conclave Sample: Database Enclave
+
+TEE like Intel SGX can be used for secure query processing.
+Sensitive data stored in the database can be encrypted when at rest or in transit.
+But this data needs to be decrypted when in use.
+When data is being used by an application it is vulnerable to a variety of attacks.
+Attacking applications while sensitive data is exposed has become a preferred strategy for security hackers looking to leverage vulnerabilities.
+With the new 3rd Gen Intel Xeon Scalable Processors supporting 1 TB of enclave memory, setting up database inside and enclave is very well
+possible.
+
+Conclave 1.2 supports persistence. File I/O operations can now be mapped to a persistent memory which is represented by a file created on the host.
+
+## Steps to enable persistence
+Add the below property to build.gradle's conclave block.
+
+    persistentFileSystemSize = "64m"
+
+Create a file for persisting the database to this file on the host device. For this sample I have created a file name scratch.txt.
+
```

Comment:
At a first reading, it seems like the users need to create the file by themselves. Maybe we should specify this better, i.e. suggesting that they need to provide the path for a file that will be used for persistence

Reply:
done